kore/test/Test/Kore/Step/Simplification/Ceil.hs

```diff
@@ -419,40 +423,6 @@ test_ceilSimplification =
                 , domainValueChild = mkStringLiteral "a"
                 }
         assertEqual "ceil(1)" expected actual
-    , testGroup "Builtin.Map"
```

Comment:
I would also test this with regular testing. The property test below is nice, but it seems to me to be somewhat complex, and having some tests where it's obvious what happens may be valuable.

Reply:
Done! And, it found a bug in the mock symbols I was using.